taca/storage/cli.py

```diff
@@ -34,13 +35,15 @@ def archive(ctx, backend, max_runs, force, compress_only):
 @click.option('-s','--site', type=click.Choice(['swestore','archive','illumina','analysis','nas','processing-server']),
               required=True, help='Site to perform cleanup')
 @click.option('-n','--dry-run', is_flag=True, help='Perform dry run i.e. Executes nothing but log')
+@click.option('--hours', type=click.INT, help="Hours to consider as thershold, this should \
+                                               not be used if '--days' of 'storage' group is used")
 @click.pass_context
-def cleanup(ctx, site, dry_run):
+def cleanup(ctx, site, dry_run, hours):
```

Comment:
connected to before, why treat hours differently? you can use the same strategy used for days. this will make code more uniform and will be easier to read and maintain it.